src/main/kotlin/com/criptext/mail/scenes/composer/ComposerController.kt

```diff
@@ -283,6 +284,7 @@ class ComposerController(private val storage: KeyValueStorage,
                 scene.dismissPreparingFileDialog()
                 model.attachments.addAll(result.remoteFiles.map { ComposerAttachment(UUID.randomUUID().toString(), it.first, it.second, model.fileKey!!) })
                 scene.notifyAttachmentSetChanged()
+                model.attachmentsChanged = true
```

Comment:
```suggestion
```
I think we better do this only when we are sure an attachment has changed, when getting a remote file, it still needs to be encrypted and uploaded to our s3 servers.